File: get_email.rb
require 'selenium-webdriver'

username = 'user'
password = 'pass'

chromedriver_path = '/Users/myuser/Desktop/chromedriver.exe'
Selenium::WebDriver::Chrome::Service.driver_path = chromedriver_path

#Selenium::WebDriver::Chrome.path = chromedriver_path
driver = Selenium::WebDriver.for :chrome
driver.navigate.to 'https://mail.protonmail.com/login'
puts driver.title
puts 'Logging in ...'

element = driver.find_element(:name, 'username')
element.send_keys username
element.submit

element = driver.find_element(:name, 'password')
element.send_keys password
element.submit

driver.quit

File: get_email.py
from selenium import webdriver

user = ''
password = ''

url = 'https://mail.protonmail.com/login'
driver = webdriver.Chrome('/Users/myuser/Desktop/chromedriver')

driver.get(url)
driver.find_element_by_id('username').send_keys(user)
driver.find_element_by_id('password').send_keys(password)
driver.find_element_by_id('login_btn').click()
#driver.send_keys(u'\ue007')